src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -770,14 +837,23 @@ def set_output_embeddings(self, new_embeddings: torch.Tensor):
     def prepare_inputs_for_generation(
```

Comment:
I think we can add a `#copied from ... llama` on this one. The base structure of this function was updated, but the copy statement will take care of it!

Reply:
should we add copied from here?